### DISCUSSION TITLE
login give 404 via localhost

### DISCUSSION THREAD

**Discussion by Juneiak** [avitorio/outstatic#160] [Q&A] [answered]
it works perfectly if login to admin panel via vercel build, but it return 404 if i try to login via localhost. I've checked everything - env is filled correct, git auth is fine too. And it is totally clear example from this repo.

vercel
<img width="1699" alt="Снимок экрана 2023-11-24 в 05 24 07" src="https://github.com/avitorio/outstatic/assets/73548051/45cb242d-0526-4ca9-9a4e-3ac8a9251df3">

local
<img width="1086" alt="Снимок экрана 2023-11-24 в 05 23 17" src="https://github.com/avitorio/outstatic/assets/73548051/77c58a81-1449-49c5-b2b3-ef808fde3b28">

local env
<img width="1564" alt="Снимок экрана 2023-11-24 в 05 24 32" src="https://github.com/avitorio/outstatic/assets/73548051/df5bee2e-8261-4dde-aa43-bad68e0881f1">

**Comment by avitorio** [comment, marked as answer]
You are missing the `outstatic` part in your localhost auth callback url: http://localhost:3000/api/outstatic/callback

I think that's the issue. Let me know if it solves it.

**Reply by Juneiak** [reply]
its work now! thanks! So there is mistake in [documentation](https://outstatic.com/docs/local-development#create-a-different-github-oauth-app)

<img width="912" alt="Снимок экрана 2023-11-25 в 00 58 14" src="https://github.com/avitorio/outstatic/assets/73548051/f89b85fc-52cc-4259-9d32-5411581b309b">

**Reply by avitorio** [reply]
Hey, it's fixed now. Thank you for flagging this!
